pandas/_libs/tslibs/timedeltas.pxd

```diff
@@ -11,7 +11,6 @@ cdef bint is_any_td_scalar(object obj)
 cdef class _Timedelta(timedelta):
     cdef readonly:
         int64_t value      # nanoseconds
-        object freq        # frequency reference
```

Comment:
i guess this is an API change, could be done via a deprecation

Reply:
if OK, i'll revert and then deprecate in a follow-up.

Reply:
+1. Yeah appears this is never set.

IMO `is_populated` should be deprecated too and made private